crates/teloxide/tests/command.rs

```diff
@@ -229,17 +229,65 @@ fn parse_named_fields() {
 #[test]
 #[cfg(feature = "macros")]
 fn descriptions_off() {
+    // FIXME: Remove `off` doc comment when `off` removed.
```

Comment:
not sure what this fixme means

Reply:
I meant that it is not important to have the off in the Help in case we do not need to test it (after removing it), but that is no problem.